snomed/com.b2international.snowowl.snomed.exporter/src/com/b2international/snowowl/snomed/exporter/model/SnomedRf2ExportModel.java

```diff
@@ -291,25 +287,14 @@ public void setRefSetsToExport(final boolean refSetsToExport) {
 	private String initExportPath() {
 		final String token;
 		if (!singleRefSetExport) {
-			token = new StringBuilder().append("SnomedCT_Release").append(getCountryAndNameSpaceElement()).append(Dates.formatByHostTimeZone(new Date(), "yyyyMMdd-HHmm")).toString();
+			token = new StringBuilder().append("SnomedCT_Release").append(namespace).append(Dates.formatByHostTimeZone(new Date(), "yyyyMMdd-HHmm")).toString();
```

Comment:
I see the following root folder inside the export archive (result of delta export using the client):
![image](https://user-images.githubusercontent.com/2010200/30649355-5b801c74-9e20-11e7-89ec-ab43f0980c4b.png)